### PR TITLE
CNV- 34883: Separate RHEL8 and 9 installations

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.14+

Issue: [CNV-34883](https://issues.redhat.com//browse/CNV-34883)

Link to docs preview: [Installing virtctl](https://67409--docspreview.netlify.app/openshift-enterprise/latest/virt/getting_started/virt-using-the-cli-tools#installing-virtctl_virt-using-the-cli-tools)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by 0xFelix** [comment]
Ah, one thing. Can we keep Linux in this headline? 

https://apinnick.github.io/previews/CNV-34883-fix-repos/virt/getting_started/virt-using-the-cli-tools.html#virt-installing-virtctl-client_virt-using-the-cli-tools

File: modules/virt-installing-virtctl-client-yum.adoc [hunk 1]
```diff
@@ -4,33 +4,24 @@
...
-# subscription-manager repos --enable cnv-4.14-for-rhel-9-x86_64-rpms
-----
-
-** {op-system-base} 8:
+. Enable the {VirtProductName} repository for your operating system by using the `subscription-manager` CLI tool:
 +
 [source,terminal]
 ----
-# subscription-manager repos --enable cnv-4.14-for-rhel-8-x86_64-rpms
+# subscription-manager repos --enable cnv-4.13-for-rhel-8-x86_64-rpms
```
Comment (dominikholler) [inline modules/virt-installing-virtctl-client-yum.adoc:21]:
```suggestion
# subscription-manager repos --enable cnv-4.14-for-rhel-8-x86_64-rpms
```

**Review by mjpytlak — COMMENTED** [review]
Peer review lgtm @apinnick

**Pull request merged**